mlibrary/pride
# LIBSEARCH-715: Removing favorites.

<details open>
<summary> :hand:&nbsp;&nbsp;<strong>Wait!</strong></summary>

Let's make sure your pull request is ready. Ask yourself...

:white_check_mark:&nbsp;&nbsp;"Did I do a good job testing my own code?"
:white_check_mark:&nbsp;&nbsp;"Is my code easy to read and understand?"
:white_check_mark:&nbsp;&nbsp;"Have I written unit tests? Do the unit tests fully cover my code?"
:white_check_mark:&nbsp;&nbsp;"Have I tested the repository's behavior to make sure my code didn't break anything?"
:negative_squared_cross_mark:&nbsp;&nbsp;"Did I include unrelated work that could be in its own pull request?"

Once you are ready, be sure to delete this `details` section before opening your pull request.
</details>

# Overview
The second part of LIBSEARCH-715.  It removes references to favorites from pride.

> This pull request resolves/closes/fixes [LIBSEARCH-715](https://mlit.atlassian.net/browse/LIBSEARCH-715).

## Anything else?
Pride should be updated before or at the same time as prejudice since it's a removing the favorites features entirely there, it's a breaking change.

## Testing
List instructions on how to test the pull request. Some examples:

- Build the repository (`npm run build`).
- Make sure the PR is consistent in repositories that use `pride`.
- _How to use the feature_.
